pkg/proxy/apis/kubeproxyconfig/validation/validation_test.go

```diff
@@ -340,22 +340,26 @@ func TestValidateKubeProxyIPTablesConfiguration(t *testing.T) {
 	}
 
 	for _, errorCase := range errorCases {
-		if errs := validateKubeProxyIPTablesConfiguration(errorCase.config, newPath.Child("KubeProxyIPTablesConfiguration")); len(errs) == 0 {
-			t.Errorf("expected failure for %s", errorCase.msg)
-		} else if !strings.Contains(errs[0].Error(), errorCase.msg) {
-			t.Errorf("unexpected error: %v, expected: %s", errs[0], errorCase.msg)
-		}
+		t.Run(errorCase.msg, func(t *testing.T) {
+			if errs := validateKubeProxyIPTablesConfiguration(errorCase.config, newPath.Child("KubeProxyIPTablesConfiguration")); len(errs) == 0 {
+				t.Errorf("expected failure for %s", errorCase.msg)
+			} else if !strings.Contains(errs[0].Error(), errorCase.msg) {
+				t.Errorf("unexpected error: %v, expected: %s", errs[0], errorCase.msg)
+			}
+		})
 	}
 }
 
 func TestValidateKubeProxyIPVSConfiguration(t *testing.T) {
 	newPath := field.NewPath("KubeProxyConfiguration")
 	testCases := []struct {
+		name      string
 		config    kubeproxyconfig.KubeProxyIPVSConfiguration
 		expectErr bool
 		reason    string
 	}{
 		{
+			name: "test1",
```

Comment:
These should get descriptive names, or at least a `// FIXME give these tests descriptive names`

(likewise in some other places. eg, `TestHasJump` in `pkg/proxy/iptables/proxier_test.go`)